src/Misc/khronosTextureContainer2.ts

```diff
@@ -72,8 +230,7 @@ export class KhronosTextureContainer2 {
      * @returns true if the data is a KTX2 file or false otherwise
      */
     public static IsValid(data: ArrayBufferView): boolean {
-        if (data.byteLength >= 12)
-        {
+        if (data.byteLength >= 12) {

```

Comment:
Can you use KTX2FileReader.IsValid?

Reply:
I can't because the ktx2Decoder module is injected in the worker thread, not in the main thread.

Reply:
Ah, okay.